lang/funcs/core/net/cidr_to_ip_test.go

```diff
@@ -0,0 +1,64 @@
+// Mgmt
+// Copyright (C) 2013-2020+ James Shubin and the project contributors
+// Written by James Shubin <james@shubin.ca> and the project contributors
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package net
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/purpleidea/mgmt/lang/types"
+)
+
+func testCidrToIP(input string, expected string) {
+
+	ip, err := CidrToIP([]types.Value{&types.StrValue{V: input}})
+	if err != nil {
+		fmt.Println("error: ", err)
+		return
+	}
+	if ip.String() != expected {
+		fmt.Printf("Invalid output, expected %v, got %v", expected, ip.String())
+		return
+	}
+}
+
+func TestCidrToIP(t *testing.T) {
+	cidrtests := []struct {
+		name     string
+		input    string
+		expected string
+	}{
+		{"normal IPv4 cidr", "10.0.0.0/32", "10.0.0.0"},
+		{"spaced IPv4 cidr ", "  10.0.0.0/32  ", "10.0.0.0"},
```

Comment:
Please add at least one non-network, /24 entry here... Basically .0's at the end are special. So having one:

192.168.42.13/24 in there would be appreciated.

Bonus: lookup what the recommend IP addresses to use in examples are (hint use search term RFC) and add one of those too. If you can't figure this out, ask me if it takes you more than 8 min.